File: temp.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
# http://www.javascriptformat.com/#jsondataurllabel
import requests ,json

u = "s"
p = ""

r = requests.post('https://build.phonegap.com/token', auth=(u, p))
x = requests.get('https://build.phonegap.com/api/v1/me', auth=(u, p))
print x.status_code

if(x.status_code == 200):
	n = x.json()
	for i in n['apps']:
		print i['private']
elif x.status_code == 401:
	print x.json()

def login():
	print "Enter Your Login Data For PhoneGap Build Service"
	u = raw_input("Your F@(king Email : ")
	while true:
		if u == "":
			u = raw_input("Opss, blank input . email bitch plz : ")
		else:
			break
	p = raw_input("Your F@(king Password : ")
	while true:
		if p == "":
			p = raw_input("Opss, blank input . password bitch plz : ")
		else:
			break
	x = requests.get('https://build.phonegap.com/api/v1/me', auth=(u, p))
	if (x.status_code == 200):
		return True
	elif x.status_code == 401:
		return False

File: temp.js
/**
 * Created by prism on 4/8/14.
 */
'use strict';

var client = require('phonegap-build-api');
var os = require('os').platform(),
    fs = require('fs'),
    home = require('homedir'),
    Table = require('cli-table'),
    color = require('cli-color'),
config = require('./auth.js');
//client.auth({ username: config.username, password: config.password }, function(e, api) {
//    // time to make requests
//    console.log(e);
//    console.log(api);
//    console.log(os);
//    console.log(process.ENV);
//});

var location = home() + "/.HirenGap";


function dir() {
    //todo windows support
    fs.exists(location, function (exists) {
        //console.log(exists ? "it's there" : "no passwd!");
        if (!exists) {
            fs.mkdirSync(location); // todo permission
        }
        fs.openSync(location + "/token", 'w');
    });
}

function checkToken(){
    fs.exists(location + '/token',function(exists){
        if(!exists){
            console.log("You are not logged in . Please Login First!");
            login();
        }
    });
}
function login() {
    dir();
    fs.exists(location + '/token', function (exists) {
        if (!exists) {
            client.auth({ username: config.username, password: config.password }, function (e, api) {
                if (e) console.log("Ops login error, error details : ", e);
                fs.writeFileSync(location + "/token", api.token);
            });
        }
    });

}

function logout() {
    fs.exists(location + '/token', function (exists) {
        if (exists) {
            fs.unlinkSync(location + '/token');
            console.log("Logged Out!");
        }
    });
}

function readToken(){
    return fs.readFileSync(location + '/token').toString();
}

function appList(){
    checkToken();
    client.auth({ token: readToken() }, function(e, api) {
        api.get('/apps', function(e, data) {
            //console.log('error:', e);
            //console.log('data:', data);
            var table = new Table();

            table.push(
                { 'Some key': (data.apps).title }
                , { 'Another key': 'Another value' }
            );

            //console.log(table.toString());
            //console.log(data.apps[0].title);
            console.log(data.apps);
        });
    });

}

appList();
//console.log(readToken());